src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  display: block;
+  width: 200px;
+  padding: 16px;
```

Comment:
you can print simply `padding: 32px 16px 16px ;` it means 32px on the top, 16px on the right and left, and 16 on the bottom